packages/space-opera/src/components/mobile_view/mobile_view.ts

```diff
@@ -70,12 +74,23 @@ export class MobileView extends LitElement {
     return this.sessionOs === 'iOS' && this.iosUrl.length <= 1;
   }
 
-  updateState(snippet: any, urls: URLs) {
+  updateState(snippet: ModelViewerSnippetState, urls: URLs) {
     this.editorUrls = urls;
     this.hotspots = snippet.hotspots;
+    for (let hotspot of this.hotspots) {
```

Comment:
fixes hotspots not showing up on mobile, need to set position/normal to Vector3Ds